Automattic/wp-calypso
# Jetpack Connect: remove requests to activate the manage module entirely

In #13205 , we conditionally activated the manage module during Jetpack connection. @johnHackworth mentioned that we could nuke this functionality from the code-base entirely, so here we go!

To test:
- get this PR running locally
- run through the test cases for Jetpack connect. see p2: P7rd6c-Me-p2
- ensure `npm run test-client /client/state/jetpack-connect` look good!
